app/views/quote_mailer/new_quote.html.erb

```diff
@@ -12,15 +12,19 @@
 
   <p>Expect one of our hard working squad members to show up within the next <b><%= t("service_expedition_times.#{@quote.service_expedition_time}") %></b> </p>
 
-  <img alt="Selected area map" src="<%= @quote.static_map_URL.html_safe %>"><br/>
+  <img alt="Selected area map" src="<%= @static_map_URL.html_safe %>"><br/>
 
   <dt>Area to be serviced:</dt>
   <dd><%= number_with_precision(@quote.area, precision: 0, delimiter: ",") %> sq ft</dd><br/>
 
   <dt>Added salt bags:</dt>
   <dd><%= @quote.salt_bags_quantity %> </dd><br/>
 
-  <dt>Total due:</dt>
+  <dt>Discounts applied:</dt>
```

Comment:
Should probably show this `dt`/`dd` only if `@quote.discount > 0`, otherwise later when/if the automatic discount is taken off, users will be getting emails saying "Discounts applied: 0%". As a matter of fact, better make sure you set `0` as the default in the DB, or that you explicitly set a value for it somehwere, otherwise this could end up with `nil * -100` which will break.